aci/data_source_aci_l3extout.go

```diff
@@ -39,27 +39,28 @@ func dataSourceAciL3Outside() *schema.Resource {
 
 			"target_dscp": &schema.Schema{
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 			},
 			"mpls_enabled": {
 				Type:     schema.TypeString,
-				Optional: true,
+				Computed: true,
+			},
+			"pim": {
+				Type:     schema.TypeList,
```

Comment:
Please add the Elem to the pim attribute.
```
				Elem: &schema.Schema{
					Type: schema.TypeString,
				},
```

Reply:
ok